Podfile

```diff
@@ -1,31 +0,0 @@
-source 'https://cdn.cocoapods.org/'
```

Comment:
We're no longer using CocoaPods for the demo app, so we can remove the Podfile and Podfile.lock.